src/index.html

```diff
@@ -1,13 +1,80 @@
+<!-- #region HEAD --------------------------->
+
 <!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+        name="viewport"
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+        rel="preconnect"
+        href="https://fonts.googleapis.com"
+    >
+    <link
+        rel="preconnect"
+        href="https://fonts.gstatic.com"
+        crossorigin="attr-req-value"
+    >
+    <link
+        href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+        rel="stylesheet"
+    >
+    <link
+        rel="stylesheet"
+        href="styles/index.scss"
+    >
   </head>
+  <!-- #endregion HEAD ---------------------->
   <body>
-    <h1>Product cards</h1>
+    <!-- #region card ----------------------->
+    <div
+        class="card"
+        data-qa="card"
+    >
+      <img
+          src="./images/imac.jpeg"
+          alt="Mac"
+          class="card__img"
+      >
```

Comment:
```suggestion
    <div
      class="card"
      data-qa="card"
    >
      <img
        src="./images/imac.jpeg"
        alt="APPLE A1419 iMac 27" Retina 5K Monoblock"
        class="card__img"
      >
```
Checklist:
13. [SEO] - alt atribute should describe the image if the image contains information (better description you have - better for you :))
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/efc1b00a-d41c-469d-99aa-404cdf0c5ea6)
14. [CODE STYLE] - If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.

You should use the same indentation style in all places.
Also, don't leave unnecessary comments.